tizen/src/urlWorker.cpp

```diff
@@ -71,6 +80,10 @@ void UrlWorker::run() {
         stream.seekp(0);
         CURLcode result = curl_easy_perform(curlHandle);
 
+        if (thread->canceled) {
```

Comment:
This is good, but we can probably also do the same check before actually making the url request through `curl_easy_perform`. Basically avoid doing a network call if that url is not required now.